docs/src/gen/consumer/file.rst

```diff
@@ -87,16 +89,24 @@ Parameters
   
   
 
-**PollingDelay**
+**PollingDelayMs** (default: 100, unit: ms)
 
-  This value defines the duration the consumer waits between
-  checking the source file for new content after hitting the end of file (EOF).
-  The value is in milliseconds (ms). NOTE: This settings only takes effect if the consumer
-  is running in `poll` mode!
+  This value defines the duration in Milliseconds the consumer
+  waits between checking the source file for new content after hitting the
+  end of file (EOF). NOTE: This settings only takes effect if the consumer is
+  running in `poll` mode!
   By default this parameter is set to "100".
   
   
 
+**RetryDelaySec** (default: 3, unit: s)
+
+  This value defines the duration in seconds the consumer waits
+  between retries, e.g. after not being able to open a file.
+  By default this parameter is set to "3".
```

Comment:
Is there a reason that values are not written as `code`? It would also be nice, to always make sure that the correct value is written inside the documentation, especially if there's a difference between `"3"` and `3`.

Reply:
that's in line with the rest of the documentation, i.e. a change would be beyond this PR

Reply:
I'm just wondering if, considering it's already inconsistent in just this single file, we should start making it consistent bit by bit while accepting inconsistency between the consistent files and the inconsistent files in the meantime. If you think that's not the right thing to do I'm also fine with leaving it the way it currently is, I just wanted us to at least consider a possible cleanup.

Reply:
If it's inconsistent in more files (I was not aware of that) my previous comment is even more true. We should find a consent here.